mackup/applications/mps-youtube.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```

Comment:
> I'm sorry for my behaviour yesterday, I had a really bad day and acted like an idiot.

No worries here, it’s all good. We disagreed, but it’s not like I thought you were being rude, or anything. The conversation remained civil, and that’s what matters.